vars/sectionNightlyTests.groovy

```diff
@@ -49,6 +49,52 @@ def call(PipelineCallbacksRunner pcr, AppPipelineConfig config, PipelineType pip
         }
       }
     }
+    if (config.parallelCrossBrowserTest) {
+      stageWithAgent("Cross browser tests", product) {
+        warnError('Failure in parallelCrossBrowserTest') {
+          Set<String> browsers = config.parallelCrossBrowsers.collect{ it.toLowerCase() }.toSet()
```

Comment:
not in use?

Reply:
yeah, I'm hoping to use it if I can get the override functionality working properly